### ISSUE TITLE
Bybit load_markets() is missing options markets

### ISSUE THREAD

**Issue by kieran-mackle** [ccxt/ccxt#18148]
### Operating System

Linux

### Programming Languages

python

### CCXT Version

3.1.28

### Description

The Bybit implementation fails to load any options markets, as it is not included in the [`fetch_markets` method](https://github.com/ccxt/ccxt/blob/master/python/ccxt/bybit.py#L1372-L1383). Attempting to place an order or even fetch data for any options contract consequently fails, with a `BadSymbol` error, as they are not in `self.markets`. The [example code provided for creating an options order](https://github.com/ccxt/ccxt/blob/cf6aa8f8125729ad3acb08bbe74416a56086123b/examples/py/bybit-USDC-create-option-order.py) does not work anymore for the same reason. 

Modifying the [`fetch_markets` method](https://github.com/ccxt/ccxt/blob/master/python/ccxt/bybit.py#L1372-L1383) (including in [async_support.bybit](https://github.com/ccxt/ccxt/blob/cf6aa8f8125729ad3acb08bbe74416a56086123b/python/ccxt/async_support/bybit.py#L1364-L1384)) to include loading `self.fetch_derivatives_markets({'category': 'option'})`, as per the code below, resolves the issue.

### Code

```python
  def fetch_markets(self, params={}):
      """
      retrieves data on all markets for bybit
      see https://bybit-exchange.github.io/docs/v5/market/instrument
      :param dict params: extra parameters specific to the exchange api endpoint
      :returns [dict]: an array of objects representing market data
      """
      if self.options['adjustForTimeDifference']:
          self.load_time_difference()
      promisesUnresolved = [
          self.fetch_spot_markets(params),
          self.fetch_derivatives_markets({'category': 'linear'}),
          self.fetch_derivatives_markets({'category': 'inverse'}),
          self.fetch_derivatives_markets({'category': 'option'}),
      ]
      promises = promisesUnresolved
      spotMarkets = promises[0]
      linearMarkets = promises[1]
      inverseMarkets = promises[2]
      optionMarkets = promises[3]
      markets = spotMarkets
      markets = self.array_concat(markets, linearMarkets)
      markets = self.array_concat(markets, optionMarkets)
      return self.array_concat(markets, inverseMarkets)
```

**Comment by carlosmiei** [comment]
Hello @kieran-mackle, thanks for letting us know,  I will restore it asap.

**Comment by carlosmiei** [comment]
@kieran-mackle This issue should be fixed by now (as of 3.1.31), but you need to set up the `defaultType` as `option` on the exchange.

Example:
```Python
    exchange = ccxt.bybit({
        'options': {
            'defaultType': 'option',
        },
    })
```

**Comment by kieran-mackle** [comment]
@carlosmiei great - thanks for the quick response!
